subsys/net/lib/sockets/sockets_tls.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2018 Intel Corporation
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <net/net_context.h>
+#include <net/socket.h>
+
+#include "sockets_tls.h"
```

Comment:
This file is not part of this commit, and following commit removes this #include (apparently, the decision was made to make "sockets_tls.h" contents inline). So, I guess it makes sense to give another thought whether it should be inline or standalone, and if the former, then remove this line right from this commit (i.e. make each commit buildable across e.g. bisecting).